CHANGELOG.txt

```diff
@@ -1,5 +1,11 @@
-4.0.2
+4.1.0
+* Exposes billing metrics
+* Adds runtime environment headers for http fauna requests
+* Always sends query timeout header
+* Enables user to pass a custom user agent
 * Adds ability to combine Fields
```

Comment:
Is there an example demonstrating how to do this?

Reply:
I got this one from this PR:
https://github.com/fauna/faunadb-jvm/commit/0ce9554e3463aad42d05b7bfbae391ad1f7c9951
but looking at this PR I don't understand why the method `at` introduced in it is about combining fields, I can be mistaken in this judgment though.
Here is an example of usage:
```scala
it should "combine fields" in {
    val data = ObjectV("data" -> pt1)

    val X = DataField.at(XField)
    val Y = DataField.at(YField)

    data(X).get shouldBe 10
    data(Y).get shouldBe 110
  }
```